packages/vow/src/vow-utils.js

```diff
@@ -38,21 +38,14 @@ const ALREADY_VOW = harden({});
  * @returns {Promise<U>}
  */
 export const unwrapPromise = async (specimenP, cb) => {
```

Comment:
I ripped out `unwrapPromise` because it was only useful when I was trying to be defensive in a couple of places.  Turns out that making `when` an async function eliminated the need.